### PR TITLE
Update sccyou

### PR DESCRIPTION
**Pull request by bturkus**
add vtt output

### CODE REVIEW

File: sccyou [hunk 1]
```diff
@@ -238,15 +230,17 @@ while [[ "${*}" != "" ]] ; do
             _report -w "${SRT_OUT} already exists"
             break
         fi
+        if [[ -f "${VTT_OUT}" ]] && [[ "${VTT_OUTPUT}"  == "y" ]] ; then
```
Comment (retokromer) [inline sccyou]: nit: there is double space before `==`

File: sccyou [hunk 2]
```diff
@@ -286,9 +280,16 @@ while [[ "${*}" != "" ]] ; do
...
     _report -d "${CC_LOG} - ${TRANSITION_TIME}."
     _end_scc
+
+    # Convert SCC to SRT if requested
     if [[ "${SRT_OUTPUT}"  == "y" ]] ; then
         "${FFMPEG_PATH}" "${FFMPEG_INPUT_OPTIONS[@]}" -i "${SCC_OUT}" "${SRT_OUT}"
     fi
 
+    # Convert SCC to WebVTT if requested
+    if [[ "${VTT_OUTPUT}"  == "y" ]] ; then
```
Comment (retokromer) [inline sccyou:299]: ditto

Comment (bturkus) [reply sccyou:299]: thanks reto, made the changes

File: sccyou [hunk 3]
```diff
@@ -24,39 +24,29 @@ $(basename "${0}") ${VERSION}
...
-   specified number of seconds may be searched to find captions. Note
-   that searching for the caption line takes time, so providing the '-l'
-   option to specify the line to use is faster.
+   The location of the captions may either be specified as a line or a
+   specified number of seconds may be searched to find captions.
 
    -l <line>    specify the line where caption data should be read,
                 otherwise the first ${SEARCH_FIRST_X_SECONDS_FOR_CAPTION_LINE} lines will be searched.
    -t <secs>    specify an amount of time in seconds for which to search
-                for captions, ${SEARCH_FIRST_X_SECONDS_FOR_CAPTION_LINE} is used by default. By default the first 20
```
Comment (dericed) [inline sccyou]: hmm, you removed the info about the default search time (20)

File: sccyou [hunk 4]
```diff
@@ -24,39 +24,29 @@ $(basename "${0}") ${VERSION}
...
-                seconds of the video is searched.
+                for captions, ${SEARCH_FIRST_X_SECONDS_FOR_CAPTION_LINE} is used by default.
 
   Outcome
-   will create MOVIE_FILE.scc and MOVIE_FILE.srt
-
-  Information about the scc format is available at http://www.theneitherworld.com/mcpoodle/SCC_TOOLS/DOCS/
```
Comment (dericed) [inline sccyou:45]: I found this helpful but ...

File: sccyou [hunk 5]
```diff
@@ -24,39 +24,29 @@ $(basename "${0}") ${VERSION}
...
+   will create MOVIE_FILE.scc and optionally MOVIE_FILE.srt or MOVIE_FILE.vtt
 
   Dependencies: ffmpeg 4.3 or later
   
  -F <path> (provide a custom ffmpeg path. ffprobe is presumed to neighbor ffmpeg)
 
-  If an ffmpeg binary is provided with the '-F <path>' option that will be used,
```
Comment (dericed) [inline sccyou]: why trash this?

File: sccyou (hunk 4 above)
Comment (retokromer) [reply sccyou:45]: I also find this useful. What about moving the info to the ReadMe file?

File: sccyou (hunk 5 above)
Comment (bturkus) [reply sccyou]: ok all good now, gonna merge

**Pull request merged**